File: CS/WebSite/App_Code/PersistentClasses.cs
using DevExpress.Xpo;

namespace DXSample {
    [Persistent("Persons")]
    public class Person :XPObject {
        public Person(Session session) : base(session) { }

        private string fName;
        public string Name {
            get { return fName; }
            set { SetPropertyValue<string>("Name", ref fName, value); }
        }
    }
}

File: VB/WebSite/App_Code/PersistentClasses.vb
Imports DevExpress.Xpo

Namespace DXSample
    <Persistent("Persons")> _
    Public Class Person
        Inherits XPObject

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub

        Private fName As String
        Public Property Name() As String
            Get
                Return fName
            End Get
            Set(ByVal value As String)
                SetPropertyValue(Of String)("Name", fName, value)
            End Set
        End Property
    End Class
End Namespace